src/generator/operations.ts

```diff
@@ -426,7 +426,16 @@ function createProtocolRequest(codeModel: CodeModel, client: string, op: Operati
     if (encodedParams.length > 0) {
       text += '\tquery := u.Query()\n';
       for (const qp of values(encodedParams)) {
-        text += emitQueryParam(qp, `query.Set("${qp.language.go!.serializedName}", ${formatParamValue(qp, imports)})`);
+        let setter: string;
+        if (qp.protocol.http?.explode === true) {
```

Comment:
For my own knowledge, what does explode mean for us in swagger?

Reply:
It comes from https://swagger.io/specification/.  Unfortunately I can't link to the exact section but if you search for "explode" it's the first hit.

Reply:
got it, thanks for the info!